grakn-dashboard/src/components/graphPage/footer/nodeLabelPanel.vue

```diff
@@ -160,7 +165,17 @@ export default {
   },
   methods: {
     configureNode(p) {
-      if (NodeSettings.getLabelProperties(this.nodeType).includes(p)) {
+      if (p == null){   
+        const nodeProps = NodeSettings.getLabelProperties(this.nodeType);
+
+        // Reset all node attributes
+        for(var i in nodeProps){
+          NodeSettings.removeTypeLabel(this.nodeType, nodeProps[i]);;
+        }   
+        // Reset color
+        visualiser.setColourOnNodeType(this.node.baseType, this.node.type);
+      }
```

Comment:
Why are we resetting also the colour?

Reply:
Given that we already have a button for resetting colour I think we should keep the functionalities separated

Reply:
So should we also have a similar button for resetting the colour since right now we have a cross on top of the colours? Or should we keep that as it is?